EmulatorPkg/EmulatorPkg.dsc

```diff
@@ -534,6 +534,7 @@
   MSFT:*_*_*_DLINK_FLAGS     = /ALIGN:4096 /FILEALIGN:4096
   MSFT:DEBUG_*_*_DLINK_FLAGS = /BASE:0x10000
   MSFT:NOOPT_*_*_DLINK_FLAGS = /BASE:0x10000
+  CLANGPDB: *_*_*_DLINK_FLAGS = /ALIGN:4096

```

Comment:
This appears to be accomplishing the same part as this PR: https://github.com/tianocore/edk2/pull/11430 was, before @deepak5x started removing pieces from that PR in favor of opening new PRs. It however has not taken the PR feedback there.

@girrajxkushwah please work with @deepak5x and only have a single PR for this change and ensure it has taken @mdkinney 's feedback from that PR into account.